benchmark/utils/main.swift

```diff
@@ -89,6 +89,7 @@ import Exclusivity
 import ExistentialPerformance
 import Fibonacci
 import FindStringNaive
+import FlattenDistanceFromTo
```

Comment:
Please also add `register(FlattenDistanceFromTo.benchmarks)` at line 286 below.
With the PR as-is the benchmark runner still does not know about this addition.

Reply:
Sure. The script could use some love too:

> The script will automatically:
>
> 3. Edit main.swift by importing and registering* your new Swift module.
>
> *The ~~cake~~ *auto-registration* is a lie.